src/components/Router/StepsRouter.tsx

```diff
@@ -59,13 +60,22 @@ class StepsRouter extends Component<StepsRouterProps> {
       ? cobrand
       : globalUserOptions.enterpriseFeatures?.cobrand && cobrand
 
+    const logoCobrandLogic = mobileFlow
+      ? logoCobrand
+      : globalUserOptions.enterpriseFeatures?.logoCobrand && logoCobrand
+
+    const logoCobrandStyle = logoCobrandLogic
+      ? { backgroundImage: 'url(' + logoCobrandLogic.src + ')' }
```

Comment:
```suggestion
      ? { backgroundImage: `url(${logoCobrandLogic.src})` }
```

Reply:
Is there any restrictions/guidelines for the type of image, dimensions that are allowed for the cobrand logo allowed/recommended? If so it should be documented in the README.

Reply:
Thanks for the format fix, I've gone ahead and updated it.

As for the image restrictions and documentation, my impression was all enterprise feature related docs should be kept out of the public README, we instead send a separate internal doc upon approval with instructions on how to enable the feature in the code after we've enabled it for their account. I've gone ahead and updated that doc to include the requirements for the image.

Reply:
Ah, ok. That's fine then. Probably just something I missed or perhaps a separate discussion to be had about letting everyone know about the internal doc for the Enterprise Features.